examples/encfs/TROUBLESHOOTING.md

```diff
@@ -14,6 +15,25 @@ Generate a new bearer token and copy it into the importkeyconfig.json.
 az account get-access-token --resource https://managedhsm.azure.net
 ```
 
+If the http response status is still 401 Unauthorized, check whether the identity you logged in has access to the AKV/mHSM you tried to import keys into. 
+
+2. 
+```
+pulling AKV response body failed: {"error":{"code":"BadParameter","message":"JSON Web Key: k property of oct key is too large, maximum size is 64 (Activity ID: 41c*****d6)"}}: http response status equal to 400 Bad Request
+
+```
+
+This might indicate that you tried to import an rsa key as an oct key. Or at least the `kty` on `importkeyconfig.json` is inconsistent with the actual key you tried to import. Make sure you configure the right `kty` on `importkeyconfig.json`. 
```

Comment:
Combine the sentences: 
This might indicate that you tried to import a rsa key as an oct key or that the `kty` on `importkeyconfig.json` is inconsistent with the actual key you tried to import.